pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -676,6 +684,16 @@ func (t *ResolvedPipelineTask) resolveTaskResources(
 ) error {
 	spec, taskName, kind, err := resolveTask(ctx, taskRun, getTask, pipelineTask)
 	if err != nil {
+		var verificationErr *trustedresources.VerificationError
+		if errors.As(err, &verificationErr) {
+			spec.SetDefaults(ctx)
+			rtr, resolveErr := resolvePipelineTaskResources(pipelineTask, &spec, taskName, kind)
+			if resolveErr != nil {
+				return fmt.Errorf("couldn't match referenced resources with declared resources: %w", resolveErr)
```

Comment:
We should reword this - e.g. Was this a client error or a server error? What should users do to resolve this?